views/js/previewer/provider/item/scss/item.scss

```diff
@@ -1,10 +1,12 @@
-@import "inc/bootstrap";
-@import "inc/colors";
-@import "inc/loading-bar";
-@import "inc/action-bars";
-@import "inc/section-container";
+@import "../../../../../../../tao/views/scss/inc/bootstrap";
+@import "../../../../../../../tao/views/scss/inc/colors";
+@import "../../../../../../../tao/views/scss/inc/loading-bar";
+@import "../../../../../../../tao/views/scss/inc/action-bars";
+@import "../../../../../../../tao/views/scss/inc/section-container";
+@import "../../../../../../../taoItems/views/scss/preview.scss";
```

Comment:
oO Is it actually needed? The SASS engine should be able to load the includes without having to apply such a redirection